packages/application/src/lab.ts

```diff
@@ -206,10 +206,18 @@ export class JupyterLab extends JupyterFrontEnd<ILabShell> {
    */
   protected evtKeydown(event: KeyboardEvent): void {
     // Process select keys which may call `preventDefault()` immediately
+    // TODO: generalise with https://github.com/jupyterlab/lumino/issues/688
     if (
+      // Navigation shortcuts which do not result in user input
+      // (except for Tab which is handled specially).
       ['Tab', 'ArrowDown', 'ArrowUp', 'ArrowRight', 'ArrowLeft'].includes(
         event.key
-      )
+      ) ||
+      // Saving shortcut which competes with the default browser action
+      (event.ctrlKey && event.key.toLowerCase() == 's') ||
```

Comment:
this not yet handle mac save file cmd+s that also got pop up browser save

Reply:
Thank you! This is now done in https://github.com/jupyterlab/jupyterlab/pull/15913/commits/ba68439adbf4cc9ad27660cad9fd6e3e5b346135.